src/frontends/tensorflow/src/decoder_proto.hpp

```diff
@@ -18,6 +18,11 @@ namespace ov {
 namespace frontend {
 namespace tensorflow {
 
+void parse_producer_name(const std::string& producer_port_name,
```

Comment:
Here is also no documentation

Reply:
Not a blocker for merge because this is internal function. If this is critical, I can fix it in separate PR.